### ISSUE TITLE
MAPS feature cannot be fetched or seen on mobile.

### ISSUE THREAD

**Issue by Fan-Karwanta** [aelassas/movinin#6]
Unlike the web app, the mobile app doesnt seems to have the mapping feature. I hope you can add this since it will be a huge advantage to have in mobile.

**Comment by Fan-Karwanta** [comment]
is this feature already added in the mobile app of movin in?

**Comment by aelassas** [comment]
No, not yet but I added property address and Google Maps link to confirmation email after successful checkout.

Now if you set property address and property coordinates in create/update property pages in the backend, the end user will receive property address and Google Maps link in the confirmation email after successful checkout.

I have not added the Map to mobile app yet for few reasons:
1. To use Google Maps in the mobile app you will have to pay Google to get API key and use Google Maps services
2. I have already added the Map to a mobile app for a customer and the setup, run from source and build instructions become complicated (for now instructions are very easy and straightforward)
3. To use Google Maps with Expo, Google Maps API Key will have to be set in app.json and since this is a public repo I don't want my API key to exposed to public (even if I can restrict API key usage to movinin mobile app SHA-1 fingerprint)

**Comment by aelassas** [comment]
I will not add this feature for the moment because run from source and build instructions will become very complicated (for now instructions are very easy and straightforward).
